paper.tex

```diff
@@ -0,0 +1,223 @@
+\documentclass[format=acmsmall]{acmart}
+\begin{document}
+\title{GPU-friendly Stroke Expansion}
+\author{Raph Levien}
+\author{Arman Uguray}
+\affiliation{
+    \institution{Google}
+    \city{San Francisco}
+    \state{CA}
+    \country{USA}
+}
+
+\begin{abstract}
+    Vector graphics includes both filled and stroked paths as the main primitives. While there have been many proposed techniques for rendering filled paths on GPU, stroked paths have proved more elusive. This paper presents a technique for performing stroke expansion, namely the generation of an outline representing the stroke of the given input path, an operation which is considered global, but which we nonetheless implement using a fully parallel algorithm. Our method relies on several novel techniques, including an encoding of the input suitable for parallel decoding and processing, an Euler spiral based method for computing flattened parallel curves. While we focus on flattened output consisting of polylines, the methods can be adapted to preserving curves in the output as well.
```

Comment:
(highlighting my suggested modifications in bold and non-italic and removals as strikethrough):

> Our method relies on several novel techniques, including an encoding of the input suitable for parallel decoding and processing, an Euler spiral based method for computing flattened parallel curves.

This sentence reads like it's incomplete. How about:

**We introduce** _several novel techniques, including an encoding of_ **stroked primitives** _suitable for parallel ~~decoding and~~ processing_ **and** _an Euler spiral based method for computing flattened parallel curves._